pulp/apis/core.py

```diff
@@ -205,6 +205,7 @@ def __init__(
         self.msg = msg
         self.options = options
         self.timeLimit = timeLimit
+        self.maxNodes = maxNodes
```

Comment:
as the comment two lines below says, the maxNodes attribute is stored inside the `self.optionsDict` attribute.